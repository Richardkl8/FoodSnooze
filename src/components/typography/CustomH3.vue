<template>
    <h3>
        <slot></slot>
    </h3>
</template>

<script>
    export default {
        name: 'CustomH3',
    };
</script>

<style scoped>
    h3 {
        font-size: 2rem;
        color: #071E22;
    }
</style>
