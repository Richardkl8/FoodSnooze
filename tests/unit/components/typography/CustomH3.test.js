import { shallowMount } from '@vue/test-utils';
import CustomH3 from '../../../../src/components/typography/CustomH3.vue';

describe('CustomH3', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(CustomH3, {
            slots: {
                default: 'Test text',
            },
        });
    });

    test('Should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('Should show text when text is injected in slot', () => {
        expect(wrapper.text()).toMatchSnapshot('Test text');
    });
});
