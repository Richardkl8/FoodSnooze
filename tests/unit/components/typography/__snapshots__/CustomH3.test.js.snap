// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomH3 Should match snapshot 1`] = `<h3>Test text</h3>`;

exports[`CustomH3 Should show text when text is injected in slot: Test text 1`] = `"Test text"`;
